multi_partner_learning.py

```diff
@@ -480,17 +524,33 @@ def collaborative_round_fit(model_to_fit, train_data, val_data, batch_size):
         """Fit the model with arguments passed as parameters and returns the history object"""
 
         x_train, y_train = train_data
-        history = model_to_fit.fit(
-            x_train,
-            y_train,
-            batch_size=batch_size,
-            epochs=1,
-            verbose=0,
-            validation_data=val_data,
-        )
-
+        if isinstance(model_to_fit, type(LogisticRegression())):
+            history = model_to_fit.fit(x_train, y_train)
```

Comment:
This does not return an history object but return the model. But maybe we should create one from scratch to mimic the output of keras ?

Reply:
I did it with a dict with a single value for 
`loss`
`accuracy`
`val_loss`
`val_accuracy`
Hope it improves compatibility